docs/broker/filtering.md

```diff
@@ -216,6 +216,24 @@ spec:
 
 ## Caveats
 
+### Variable prefixes
+
+Due to limitations in the CloudEvents SDK, dynamic fields like extensions and
```

Comment:
@n3wscott remind me to discuss this with you at your convenience.

Reply:
the expression can act on the json though right? Lets chat.